network/network.go

```diff
@@ -228,9 +263,10 @@ func (n *network) AppRequest(ctx context.Context, nodeID ids.NodeID, requestID u
 
 	log.Debug("received AppRequest from node", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request))
 
-	if !IsNetworkRequest(requestID) {
-		log.Debug("forwarding AppRequest to SDK network", "nodeID", nodeID, "requestID", requestID, "requestLen", len(request))
-		return n.p2pNetwork.AppRequest(ctx, nodeID, requestID, deadline, request)
+	var req message.Request
+	if _, err := n.codec.Unmarshal(request, &req); err != nil {
```

Comment:
This change overrides this [PR](https://github.com/ava-labs/subnet-evm/pull/1593) from [Coreth PR](https://github.com/ava-labs/coreth/pull/977) from